ci/travis/test-wheels.sh

```diff
@@ -45,15 +46,18 @@ if [[ "$platform" == "linux" ]]; then
 
   # Run a simple test script to make sure that the wheel works.
   INSTALLED_RAY_DIRECTORY=$(dirname "$($PYTHON_EXE -u -c "import ray; print(ray.__file__)" | tail -n1)")
-  $PYTHON_EXE "$TEST_SCRIPT"
+
```

Comment:
@mehrdadn I did something super lazy here. It turns out we are not testing all Python version in linux build. Can you add that here? Thanks!